src/axom/mint/tests/mint_execution_cell_traversals.cpp

```diff
@@ -30,6 +30,22 @@ namespace mint
 //------------------------------------------------------------------------------
 namespace
 {
+#if defined(AXOM_USE_RAJA) && defined(AXOM_USE_UMPIRE) &&   \
```

Comment:
It looks like all this code is the same as the code in the files for mint tests of face traversals and node traversals. It seems like it would save a lot of maintenance if it were centralized in one place and reused. @bmhan12 if you agree, this could be done in a subsequent PR.

Reply:
Added this as a task part of #1339